perfect_slice_share.py

```diff
@@ -0,0 +1,20 @@
+# function to check if the number is a perfect slice share
```

Comment:
I think that I have added comments before you remerged it.
The comments that I give you are pretty repetitive - You must describe more in your documentations, and document inside the function. Dont forget to describe your parameters, and give them more informative names (avoid single letter names).